index.html

```diff
@@ -0,0 +1,152 @@
+                        Learn More <img src="./images/Vector.png">
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+    <div class="teams">
+        <div class="teams__text">
+            <label class="welcome__text--padded welcome__text--blue">
+                Team
+            </label>
+            <label class="content__text--big">Our Team</label>
+            <div class="content__text--muted">
+                Problems trying to resolve the conflict between 
+                the two major realms of Classical physics: Newtonian mechanics 
+            </div>
+        </div>
+        <div class="teams__images">
+            <div class="teams__div">
+                <div class="card">
+                    <img class="card__image" src="./images/doc1.png">
+                    <label class="card__name text--bold">
+                        Julian Jameson
+                    </label>
+                    <label class="card__proffesion text--muted">
+                        Proffesion
+                    </label>
+                    <div class="card__socials">
+                        <img src="./images/fb.png">
+                        <img src="./images/ig.png">
+                        <img src="./images/tw.png">
+                    </div>
+                </div>
+                <div class="card">
+                    <img class="card__image" src="./images/doc2.png">
+                    <label class="card__name text--bold">
+                        Julian Jameson
+                    </label>
+                    <label class="card__proffesion text--muted">
+                        Proffesion
+                    </label>
+                    <div class="card__socials">
+                        <img src="./images/fb.png">
+                        <img src="./images/ig.png">
+                        <img src="./images/tw.png">
+                    </div>
+                </div>
+            </div>
+            <div class="teams__div">
+                <div class="card">
+                    <img class="card__image" src="./images/doc3.png">
+                    <label class="card__name text--bold">
+                        Julian Jameson
+                    </label>
+                    <label class="card__proffesion text--muted">
+                        Proffesion
+                    </label>
+                    <div class="card__socials">
+                        <img src="./images/fb.png">
+                        <img src="./images/ig.png">
+                        <img src="./images/tw.png">
+                    </div>
+                </div>
+                <div class="card">
+                    <img class="card__image" src="./images/doc4.png">
+                    <label class="card__name text--bold">
+                        Julian Jameson
+                    </label>
+                    <label class="card__proffesion text--muted">
```

Comment:
It should be `card__profession card__profession--text-muted`

Reply:
Thanks for niticing